interface/forms/ros/C_FormROS.class.php

```diff
@@ -1,30 +1,47 @@
 <?php
-
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+ 
 require_once ($GLOBALS['fileroot'] . "/library/classes/Controller.class.php");
 require_once ($GLOBALS['fileroot'] . "/library/forms.inc");
 require_once("FormROS.class.php");
 
 class C_FormROS extends Controller {
 
-	var $template_dir;
+	public $template_dir;
 	
-    function __construct($template_mod = "general") {
+    public function __construct($template_mod = "general") {
     	parent::__construct();
-    	$returnurl = 'encounter_top.php';
+    	$returnurl = $GLOBALS['concurrent_layout'] ? 'encounter_top.php' : 'patient_encounter.php';
```

Comment:
Just means your code is a bit out of date. The concurrent_layout was stripped out of the codebase a couple weeks ago. Will need to change it to:
``````
$returnurl = 'encounter_top.php';